### PR TITLE
Feat/appeal model dto

### PR DESCRIPTION
**Pull request by fahmirzldi**
add appeal model and dto

### CODE REVIEW

File: app/main/model/appeal.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import uuid
+import datetime
+from .. import db
+from ..util.helper import convert_to_local_time
+# from .Report import Report
```
Comment (fatjan) [inline app/main/model/appeal.py:5]: please remove this commented line

File: app/main/model/appeal.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        return f"<Appeal(reason={self.reason})>"
+
+    def serialize(self):
+        created_at = convert_to_local_time(self.created_at)
+        updated_at = convert_to_local_time(self.updated_at)
+        # report_model = Report()
+        # report_id = report_model.get_report_public_id(self.report_id)
+        return {
+            "public_id": self.public_id,
+            # 'user_id': self.user_id,
```
Comment (fatjan) [inline app/main/model/appeal.py:29]: you will need to use the public_id of user_id as well later

**Review by fatjan — DISMISSED** [review]

**Comment by fahmirzldi** [comment]
Solve conflict, please kindly accept PR

**Review by fatjan — DISMISSED** [review]

**Comment by fatjan** [comment]
why is this PR only have the migration changes?

**Comment by fahmirzldi** [comment]
It because the create appeal PR which include the model and dto already merged, should I drop this PR?

**Comment by fatjan** [comment]
> It because the create appeal PR which include the model and dto already merged, should I drop this PR?

So, this migration should have been included in the previously merged PR?
Ok, I'll approve this and you can merge it afterwards.  Make sure to include db migrations version in the same PR of your code changes in the next PRs.

**Review by fatjan — APPROVED** [review]

**Pull request merged**